go.mod

```diff
@@ -3,107 +3,103 @@ module github.com/openshift/certman-operator
 go 1.17
```

Comment:
Since boilerplate supports Go 1.19 any reason not to bump this too? I'm ok with separating it into a separate PR too if we're just trying to break apart changes in to smaller pieces.

Reply:
Yeah let's do that in a follow up. Thanks for pointing it out

Reply:
On second thought, let's do it.